conda-store-server/conda_store_server/server/auth.py

```diff
@@ -436,7 +436,9 @@ async def post_login_method(
             samesite="strict",
             domain=self.cookie_domain,
             # set cookie to expire at same time as jwt
-            max_age=(authentication_token.exp - datetime.datetime.utcnow()).seconds,
+            max_age=int(
+                (authentication_token.exp - datetime.datetime.utcnow()).total_seconds()
+            ),
```

Comment:
Why was this change necessary?

Reply:
I explained this in the commit message [`877f816` (#559)](https://github.com/conda-incubator/conda-store/pull/559/commits/877f8167b4251fae48caa814668c9ef74b7f9f18)

Reply:
Oh, good catch! This can actually fail to consider some timedelta data, too:

```py
In [10]: import datetime
    ...:
    ...: t = datetime.timedelta(hours=24, minutes=2)

In [11]:

In [11]: t.seconds  # only considers minutes after 24 hours
Out[11]: 120

In [12]: int(t.total_seconds())  # considers total seconds in timedelta
Out[12]: 86520
```